src/components/rk-date.vue

```diff
@@ -202,8 +203,9 @@ export default {
       this.show = this.$el.contains(e.target) && !this.disabled;
     },
     quickPick(type){
-      const end = new Date();
-      const start = new Date();
+      const utc = window.localStorage.getItem('utc') || 8; //默认东8区
```

Comment:
Remove Chinese character, this is not allowed in Apache repo. And don't put the default UTC timezone, keep it strictly as same as the page shows.

Reply:
ok